cmd/build/cluster.go

```diff
@@ -313,6 +313,7 @@ func (c *Cluster) loadImages(ctx context.Context, registryPort int32) error {
 		run.Fn3(pushImage, "package-operator-webhook", registry, runtime.GOARCH),
 		run.Fn3(pushImage, "remote-phase-manager", registry, runtime.GOARCH),
 		run.Fn3(pushImage, "test-stub", registry, runtime.GOARCH),
+		run.Fn3(pushImage, "test-stub-mirror", registry+"/src", runtime.GOARCH),
```

Comment:
Can we just use the `test-stub` image here and not create a new one? Then we wouldn't need `cmd/test-sub-mirror/` and `config/images/test-stub-mirror.Containerfile`.
```suggestion
		run.Fn3(pushImage, "test-stub", registry+"/src", runtime.GOARCH),
```

Reply:
Yep, sounds good. :+1: